docs/imagecreator/quick-start/quick-start.md

```diff
@@ -92,17 +92,23 @@ nav_order: 1
 
 3. Add executable permissions using `chmod +x imagecreator`.
 
-4. Download the tools file 
-   
+4. Download the tools file.
+
    For creating a 2.0 tools file run
-    ../../toolkit/tools/internal/testutils/testrpms/create-tools-file.sh
+
+    ```bash
+    ../../toolkit/tools/internal/testutils/testrpms/create-tools-file.sh \
     "mcr.microsoft.com/cbl-mariner/base/core:2.0" "tools.tar.gz"
```

Comment:
It would be good to tab in this line to make it clear it is a continuation.

Same thing below.